src/app/components/history-table/history-table.component.html

```diff
@@ -24,68 +24,58 @@
                     fxLayoutAlign="center center"
                 >
                     <mat-icon
-                        *ngIf="
-                            event.event === 'EventPaymentReceivedSuccess';
-                            else sent
-                        "
+                        *ngIf="isReceivedEvent(event); else sent"
                         class="icon-box__icon"
                         svgIcon="received"
-                        matTooltip="Received transfer"
                     ></mat-icon>
                     <ng-template #sent>
                         <mat-icon
                             class="icon-box__icon"
                             svgIcon="sent"
-                            matTooltip="Sent transfer"
                         ></mat-icon>
                     </ng-template>
                 </div>
             </cdk-cell>
         </ng-container>
 
-        <ng-container cdkColumnDef="label">
-            <cdk-cell class="table__element" *cdkCellDef="let event">
-                {{ addressLabel(paymentPartner(event)) }}
-            </cdk-cell>
-        </ng-container>
-
-        <ng-container cdkColumnDef="partner">
-            <cdk-cell
-                class="table__element table__address"
-                *cdkCellDef="let event"
-            >
-                {{ paymentPartner(event) }}
-            </cdk-cell>
-        </ng-container>
-
-        <ng-container cdkColumnDef="date">
-            <cdk-cell class="table__element" *cdkCellDef="let event">
-                {{ getUTCTimeString(event) | date: 'dd-MM-yyy H:mm' }}
-            </cdk-cell>
-        </ng-container>
-
-        <ng-container cdkColumnDef="icon">
-            <cdk-cell *cdkCellDef="let event">
-                <span *ngIf="event.userToken">
-                    <div
-                        class="icon-box icon-box--round"
-                        fxLayout="column"
-                        fxLayoutAlign="center center"
-                        matTooltip="{{ event.userToken.name }}
-                            {{ event.userToken.address }}
-                            (Click to copy)"
+        <ng-container cdkColumnDef="main">
+            <cdk-cell *cdkCellDef="let event" class="table__main">
+                <div class="table__partner">
+                    <ng-container *ngIf="isReceivedEvent(event); else sent_to"
+                        >Received from</ng-container
```

Comment:
Same here, usually for there it makes sense to add a whitespace before and after the text. There should be a lint rule.